examples/demo/site/modules/auth/scripts/auth_temp_users.sql

```diff
@@ -0,0 +1,13 @@
+
+DROP TABLE IF EXISTS "auth_temp_users";
+CREATE TABLE `auth_temp_users` (
+  `uid` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
```

Comment:
I think it should be AUTO_INCREMENT

remember, it will be handled by ROC CMS to be portable among SQL db ...
but the reference is using AUTO_INCREMENT

Check the node.sql for example.